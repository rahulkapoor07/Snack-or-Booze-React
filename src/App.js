import React from "react";
import Cafe from "./Cafe";
import NavBar from "./NavBar";
import Routes from "./Routes";
import "./App.css";

function App() {
  return (
    <div className="App">
      <Cafe>
      <NavBar />
      <Routes />
        </Cafe>
    </div>
  )}

export default App;
