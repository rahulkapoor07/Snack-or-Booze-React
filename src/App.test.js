import React from "react";
import {MemoryRouter} from "react-router-dom";
import { render } from '@testing-library/react';
import App from "./App";

describe("App page testing", ()=>{
    test("rendering", ()=>{
        render(<MemoryRouter><App /></MemoryRouter>);
    });

    test("snapshot", ()=>{
        const {asFragment} = render(<MemoryRouter><App /></MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});