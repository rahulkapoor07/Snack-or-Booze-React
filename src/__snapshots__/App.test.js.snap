// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App page testing snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <p>
      Loading â€¦
    </p>
  </div>
</DocumentFragment>
`;
